docs/validator/fullnode.md

```diff
@@ -64,6 +69,19 @@ mv server/data-seed/geth/chaindata node/geth/triecache
 
 :::
 
+:::note
+If you can not download the chaindata snapshot and want to sync from genesis, then you have to generate the genesis block first, which you have already get the genesis.json in Step 2. So just run:
+
+```
+## It will init genesis with Hash-Base Storage Scheme by default.
+geth --datadir <datadir> init ./genesis.json
+
+## It will init genesis with Path-Base Storage Scheme.
+geth --datadir <datadir> --state.scheme path init ./genesis.json
+```
+
+:::
+
 4. Start a full node
```

Comment:
rm the genesis sync and add one more line here for pbss mode, just like fastnode.

Reply:
Done. For PBSS, it's been added over here: https://github.com/bnb-chain/bnb-chain.github.io/pull/242/files#diff-b75c5258b06309047c354bfa8b5b8e37b6ba516a49aa0f48bdd158538cac05b4R113.